packages/shared/src/components/checklist/ChecklistStep.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { IconSize } from '../Icon';
+import ArrowIcon from '../icons/Arrow';
+import { ChecklistStepProps } from '../../lib/checklist';
+import { QuaternaryButton } from '../buttons/QuaternaryButton';
+import ChecklistAIcon from '../icons/ChecklistA';
+
+const ChecklistStep = ({
+  className = {},
+  step,
+  checked = false,
+  active = false,
+  onToggle,
+  children,
+}: ChecklistStepProps): ReactElement => {
+  const isCompleted = !!step.action.completedAt;
+  const isOpen = !isCompleted && checked;
```

Comment:
Does this one mean every non completed step is open?
I think the idea is that only one step at the time will be active.

Reply:
No, just a clunky naming, refactored in 9409c10e30a4dd22875ead5299d74669e3276f03